src/_components/header-search-box.component.html

```diff
@@ -0,0 +1,19 @@
+<form class="box-search">
+    <div class="udlite-search-form">
+        <label for="<!--@@var nameSearch -->" class="<!--@@var linkClass -->">Search Google:</label>
```

Comment:
Do we have class default for label?
If we have, please provide it.
example: `form-label`